app.py

```diff
@@ -0,0 +1,332 @@
+from flask import Flask, render_template,request,redirect
+import re
+import requests
+from logging.handlers import RotatingFileHandler
+import logging
+from datetime import date
+
+server="127.0.0.1"
+#server="https://ur-opinion.herokuapp.com/"
+def test_local_server():
```

Comment:
Can you please explain, what we need this function?

Reply:
We need it temporarily to prevent others access to sub_routes which are called to load into internal div.